qiita_db/handlers/artifact.py

```diff
@@ -69,6 +69,46 @@ def get(self, artifact_id):
 
         self.write(response)
 
+    @authenticate_oauth
+    def patch(self, artifact_id):
+        """Patches the filepaths of the artifact
+
+        Parameter
+        ---------
+        artifact_id : str
+            The id of the artifact whole filepaths information is being updated
+
+        Returns
+        -------
+        dict
+            Format:
+            {'success': bool,
```

Comment:
If we are going to change the API regardless, I'm fine and I think we already have an issue open, I've added a note there to make sure we don't forget about this.